.circleci/config.yml

```diff
@@ -70,28 +69,6 @@ jobs:
             - dist
             - .cache/Cypress
 
-  check_outdated:
```

Comment:
There is a version from the orb that we can use, which means we don't need to use the slack orb